### ISSUE TITLE
[Feature Request] Adding group/scene last update values to a table (in LUA)

### ISSUE THREAD

**Issue by bertoost** [domoticz/domoticz#4993]
Hi,

I would like to have an ability to read the lastupdate datetime from any group/scene. Like the `otherdevices_lastupdate`.

Currently I am storing this value in a user variable, but that is far from ideal, since the value is already stored in the Domoticz database.

Regards, Bert

**Comment by kiddigital** [comment]
Can you provide maybe a (few) example(s) use-case(s)? Just curious.

Is there maybe already a command that you are using that could logically be extended with this info?

**Comment by bertoost** [comment]
As I am using LUA to script my automation I am turning lights on when coming home (based on my phone and sunset status) via a group. Sometimes I leave immediately again to do some grocery shopping. I do not want a disco, so I built in a time-thresshold to not turn it off immediately when it's recently turned on automatically.
Thats why I would like to know the lastupdate from group/scenes.

**Comment by gizmocuz** [comment]
You are able to do this with dzVents a more advanced lua scripting language

https://domoticz.com/forum/viewtopic.php?t=20506

**Comment by bertoost** [comment]
Yeah but that's not my feature request... I have a lot of scripting in LUA and I'm missing that in LUA. Not dzVents.

**Comment by gizmocuz** [comment]
Sometimes you need more advanced scripting...
